packages/mui-material/src/Button/Button.spec.tsx

```diff
@@ -80,8 +80,8 @@ const ButtonTest = () => (
         TestOverride
       </Button>
     }
-    <Button startIcon={FakeIcon}>Start Icon</Button>
-    <Button endIcon={FakeIcon}>endIcon</Button>
+    <Button startIcon={<FakeIcon />}>Start Icon</Button>
```

Comment:
`Button` requires a `ReactNode` but `FakeIcon` is a React component. This only worked because `ReactNode` includes `{}`